### DISCUSSION TITLE
Migrating from vuex-orm and VuexORMLocalForage plugin #1534

### DISCUSSION THREAD

**Discussion by jpmc3630** [CodeDredd/pinia-orm#195] [Q&A] [answered]
Unanswered
[jpmc3630](https://github.com/jpmc3630) asked this question in [Q&A](https://github.com/vuejs/pinia/discussions/categories/q-a)

[jpmc3630](https://github.com/jpmc3630)
[5 days ago](https://github.com/vuejs/pinia/discussions/1534#discussion-4293048)
We are hoping to migrate to pinia and pinia-orm, but we are currently using the VuexORMLocalForage plugin (with vuex-orm and vuex3).

I can't find any info about pinia-orm and localforage. Is it possible to use the indexdb instead of local storage with pinia-orm?

**Comment by CodeDredd** [comment, marked as answer]
I think it should be possible. try to use https://github.com/iendeavor/pinia-plugin-persistedstate-2.
I made you an simple example:
https://codesandbox.io/s/wispy-shape-516trq?file=/src/App.vue

**Reply by jpmc3630** [reply]
Ok thanks, we will look into that plugin
